lib/features/pokemon_details/pokemon_details_page.dart

```diff
@@ -0,0 +1,98 @@
+import 'package:flutter/material.dart';
+import 'package:pokedexbootcamp/api/model/pokemon_detail.dart';
+import 'package:pokedexbootcamp/utils/async.dart';
+import 'package:pokedexbootcamp/utils/constants.dart';
+import 'package:pokedexbootcamp/utils/theme.dart';
+import 'package:pokedexbootcamp/widgets/image_widget_pokemon_details.dart';
+import 'package:pokedexbootcamp/widgets/row_description.dart';
+
+class PokemonDetailsPage extends StatefulWidget {
+  const PokemonDetailsPage({
+    Key? key,
+    required this.pokemonDetail,
+    required this.pokemonName,
+  }) : super(key: key);
+
+  final Async<PokemonDetail?> pokemonDetail;
```

Comment:
why is the PokemonDetail nullable? 🙂

Reply:
oh thats strange, I forgot to add the updated one to the commit will make it to the next push